elements/pfe-button/demo/pfe-button.html

```diff
@@ -5,175 +5,90 @@
 
 <section>
   <h2 slot="header">Default - Medium size</h2>
-  <pfe-button>
-    <button>Primary</button>
-  </pfe-button>
-  <pfe-button variant="secondary">
-    <button>Secondary</button>
-  </pfe-button>
-  <pfe-button variant="link" icon="link" icon-position="right">
-    <button>Link</button>
-  </pfe-button>
-  <pfe-button variant="link" inline><button>Inline Link</button></pfe-button>
-  <pfe-button variant="link" danger><button>Danger Link</button></pfe-button>
+  <pfe-button>Primary</pfe-button>
+  <pfe-button variant="secondary">Secondary</pfe-button>
+  <pfe-button variant="link" icon="link" icon-position="right">Link</pfe-button>
+  <pfe-button variant="link" inline>Inline Link</pfe-button>
+  <pfe-button variant="link" danger>Danger Link</pfe-button>
 
   <h3>Plain button</h3>
   <pfe-button plain>
-    <button aria-label="Action">
-      <svg fill="currentColor" height="1em" width="1em" viewBox="0 0 352 512">
-        <path d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"></path>
-      </svg>
-    </button>
+    <svg aria-label="Action" fill="currentColor" height="1em" width="1em" viewBox="0 0 352 512">
+      <path d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"></path>
+    </svg>
   </pfe-button>
```

Comment:
Use offscreen text and aria-hidden on SVG. See https://www.sarasoueidan.com/blog/accessible-icon-buttons/